platform/macos/src/MGLMapView.h

```diff
@@ -10,6 +10,7 @@ NS_ASSUME_NONNULL_BEGIN
 @class MGLAnnotationImage;
 @class MGLMapCamera;
 @class MGLStyle;
+@class MGLShape;
```

Comment:
Can you make these changes to the iOS implementation of MGLMapView also? Thanks!